### DISCUSSION TITLE
Adding an extra Kustomization in gotk-sync.yaml

### DISCUSSION THREAD

**Discussion by rombert** [fluxcd/flux2#1967] [Q&A] [answered]
Hi,

I tried to add a second Kustomization to my gotk-sync.yaml file, but that got reverted after i tore down the cluster and boostrapped it again. Is this expected? If so, what is the canonical way of adding a single extra Kustomization?

**Comment by stefanprodan** [comment, marked as answer]
Please see here an example repo https://github.com/fluxcd/flux2-kustomize-helm-example/tree/main/clusters/staging

**Reply by rombert** [reply]
Thanks for the answer. What do you think about adding a comment to the files that will always be overwritten? Something along the lines of

```yaml
# Do not make changes to this file, it will be overwritten if you run flux bootstrap again 
```

**Reply by stefanprodan** [reply]
Yes that would be great! Can you please open an issue in flux2 repo for this? Thanks

**Reply by rombert** [reply]
https://github.com/fluxcd/flux2/issues/2049 . Thanks!
